src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Logging/Log4netWrapper.cs

```diff
@@ -46,9 +46,9 @@ private void RecordLogMessage(object logEvent, IAgent agent)
             var getRenderedMessageFunc = _getRenderedMessage ??= VisibilityBypasser.Instance.GeneratePropertyAccessor<string>(logEvent.GetType(), "RenderedMessage");
             var renderedMessage = getRenderedMessageFunc(logEvent);
 
-            // We can either get this in Local or UTC
-            var getTimestampFunc = _getTimestamp ??= VisibilityBypasser.Instance.GeneratePropertyAccessor<DateTime>(logEvent.GetType(), "TimeStampUtc");
-            var timestamp = getTimestampFunc(logEvent);
+            // Older versions of log4net only allow access to a timestamp in local time
+            var getTimestampFunc = _getTimestamp ??= VisibilityBypasser.Instance.GeneratePropertyAccessor<DateTime>(logEvent.GetType(), "TimeStamp");
```

Comment:
This is such an awesome find! It really opens up the amount of versions we can support.